<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformationWidget</class>
 <widget class="QDockWidget" name="TransformationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Affine Transformation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Quick Rotation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QPushButton" name="rotX180">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsX180.png</normaloff>:/cgal/icons/resources/RotationsX180.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="rotY270">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsY270.png</normaloff>:/cgal/icons/resources/RotationsY270.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="rotX90">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsX90.png</normaloff>:/cgal/icons/resources/RotationsX90.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="rotY180">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsY180.png</normaloff>:/cgal/icons/resources/RotationsY180.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="rotX270">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsX270.png</normaloff>:/cgal/icons/resources/RotationsX270.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="rotY90">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsY90.png</normaloff>:/cgal/icons/resources/RotationsY90.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="rotZ270">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsZ270.png</normaloff>:/cgal/icons/resources/RotationsZ270.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="rotZ180">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsZ180.png</normaloff>:/cgal/icons/resources/RotationsZ180.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="rotZ90">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../Polyhedron_3.qrc">
              <normaloff>:/cgal/icons/resources/RotationsZ90.png</normaloff>:/cgal/icons/resources/RotationsZ90.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>64</width>
              <height>64</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Transformation</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Transformation Matrix</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="3" column="3">
              <widget class="QLineEdit" name="matrix_33"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="matrix_01"/>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="matrix_03"/>
             </item>
             <item row="0" column="0">
              <widget class="QLineEdit" name="matrix_00"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="matrix_31"/>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="matrix_22"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="matrix_21"/>
             </item>
             <item row="2" column="0">
              <widget class="QLineEdit" name="matrix_20"/>
             </item>
             <item row="1" column="3">
              <widget class="QLineEdit" name="matrix_13"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="matrix_11"/>
             </item>
             <item row="3" column="2">
              <widget class="QLineEdit" name="matrix_32"/>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="matrix_02"/>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="matrix_23"/>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="matrix_12"/>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="matrix_30"/>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="matrix_10"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="resetMatrix_Button">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="applyMatrix_Button">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Individual Transformation:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QComboBox" name="transfo_ComboBox">
               <item>
                <property name="text">
                 <string>Rotation: </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Translation: </string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Scaling:</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Normalize</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditA">
               <property name="toolTip">
                <string>Angle value in degrees.</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>A</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditX">
               <property name="toolTip">
                <string>Value along the x-axis.</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="placeholderText">
                <string>X</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditY">
               <property name="toolTip">
                <string>Value along the y-axis.</string>
               </property>
               <property name="placeholderText">
                <string>Y</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditZ">
               <property name="toolTip">
                <string>Value along the z-axis.</string>
               </property>
               <property name="placeholderText">
                <string>Z</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="applyTransfo_Button">
               <property name="text">
                <string>Apply</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Polyhedron_3.qrc"/>
 </resources>
 <connections/>
</ui>
